Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -195,7 +188,7 @@ @interface ConversationViewController () <AttachmentApprovalViewControllerDelega
 @property (nonatomic, readonly) UILabel *backButtonUnreadCountLabel;
 @property (nonatomic, readonly) NSUInteger backButtonUnreadCount;
 
-@property (nonatomic) NSUInteger page;
+@property (nonatomic) NSUInteger lastRangeLength;
```

Comment:
It's much simpler and more expressive to represent the size of the current "desired" range using the unit of items, not pages of items.